### PR TITLE
LPS-125846 Grey bars of assets with just one vocabulary categories

### PR DESCRIPTION
**Pull request by dgarciasarai**
**Description**

The Content Dashboard shows assets with just one vocabulary category selected. If you configure the chart with two vocabularies, we can see sections with "No XXX Specified".

**Solution**

* Use grey color for bars of asset with just one vocabulary (instead of one of the list)
* Change copy from "No {CategoryName} Category" to "No {CategoryName} Specified"

**Screenshoot**

Screenshot 2021-01-26 at 10 08 06

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

File: modules/apps/content-dashboard/content-dashboard-web/test/js/components/AuditBarChart.js [hunk 1]
```diff
@@ -177,6 +177,47 @@ const mockTwoVocabulariesWithCategoriesInTheFirstVocabulary = [
...
+				vocabularyName: 'Stage',
+			},
+			{
+				key: 'selection',
+				name: 'Selection',
+				value: 1055,
+				vocabularyName: 'Stage',
+			},
+			{
+				key: 'none',
```
Comment (darquesdev) [inline modules/apps/content-dashboard/content-dashboard-web/test/js/components/AuditBarChart.js:196]: Have you tested not using the last position in the mocked data?

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -17,7 +17,7 @@ in-use=In Use
 javax.portlet.title.com_liferay_content_dashboard_web_portlet_ContentDashboardAdminPortlet=Content Dashboard
 languages-translated-into=Languages Translated Into
 metrics=Metrics
-no-x-category=No {0} Category
+no-x-specified=No {0} Specified
```
Comment (darquesdev) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/content/Language.properties:20]: let's see if brian likes approves this statement

File: modules/apps/content-dashboard/content-dashboard-web/test/js/components/AuditBarChart.js (hunk 1 above)
Comment (dgarciasarai) [reply modules/apps/content-dashboard/content-dashboard-web/test/js/components/AuditBarChart.js:196]: Yes, the test doesn't assert the sorting, it only checks if the none name is displayed in the UI: "No Stage Specified".

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/content/Language.properties (hunk 2 above)
Comment (dgarciasarai) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/content/Language.properties:20]: Agree. I don't know...

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Review by darquesdev — APPROVED** [review]
LGTM!

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by darquesdev** [comment]
ci:forward

**Comment by cgoncas** [comment]
ci:reopen

**Comment by cgoncas** [comment]
Hey @darquesdev and @dgarciasarai !!!

Better if we follow our protocol and this pull is reviewed by the frontend team, until we find a frontend team member to review it :)

Thanks!!!

**Comment by darquesdev** [comment]
Sure @cgoncas, you're right, let's follow our guidelines ;)

**Comment by dgarciasarai** [comment]
Sure! I'll do it right now 😄

**Comment by dgarciasarai** [comment]
Continues in https://github.com/liferay-frontend/liferay-portal/pull/732.

**Pull request closed without merging**